packages/imitation_game/last_results.json

```diff
@@ -0,0 +1,14 @@
+{
+  "smiley": {
+    "flutter": {
+      "startupTime": 0.691717,
+      "adb_memory_total": 101928.0
+    },
+    "uikit": {
+      "startupTime": 0.2632870674133301
+    },
+    "android": {
+      "adb_memory_total": 35267.0
```

Comment:
how come Android doesn't have a startup time?

Reply:
No one implemented it.